eagle-hadoop-metric/src/main/webapp/app/apps/hadoop_metric/style/index.css

```diff
@@ -59,3 +59,13 @@
 .hadoopMetric-widget .hadoopMetric-chart-container {
 	height: 100px;
 }
+
+.summary-a {
+	color: #fff;
+}
+
+@media only screen and (min-device-width: 1280px) and (max-device-width: 1360px) {
```

Comment:
Make sure you need this to set the grid style(which may cause browser compatibility issue).
This will affect all the eagle application. Pls use `.hadoopMetric-` prefix.